app/build.gradle

```diff
@@ -55,7 +55,7 @@ dependencies {
         exclude module: 'support-annotations'
     }
 
-    implementation 'com.github.TeamNewPipe:NewPipeExtractor:1c97da8b51b3610'
+    implementation 'com.github.wb9688:NewPipeExtractor:6453805288'
```

Comment:
I guess you didn't want to commit this 😆

Reply:
Yeah, but https://github.com/TeamNewPipe/NewPipeExtractor/pull/60 isn't merged yet.